print.css

```diff
@@ -0,0 +1,118 @@
+@charset "UTF-8";
+/*
+Theme Name: Twenty Twenty
+Adding print support. The print styles are based on the the great work of
+Andreas Hecht in https://www.jotform.com/blog/css-perfect-print-stylesheet-98272/.
+*/
+/*--------------------------------------------------------------
+>>> TABLE OF CONTENTS:
+----------------------------------------------------------------
+# Margins
+# Typography÷
+# Page breaks
+# Links
+# Visibility
+--------------------------------------------------------------*/
+@media print {
+  /* Margins */
+  @page {
+    margin: 2cm;
+  }
+  .entry {
+    margin-top: 1em;
+  }
+  .entry-header,
+  .site-footer {
+    margin: 0;
+  }
+  /* Fonts */
+  body {
+    font: 13pt Georgia, "Times New Roman", Times, serif;
+    line-height: 1.3;
+    background: #fff !important;
+    color: #000;
+  }
+  h1 {
+    font-size: 24pt;
+  }
+  h2,
+  h3,
+  h4,
+  .has-regular-font-size,
+  .has-large-font-size,
+  .comments-header,
+  h3 {
+    font-size: 14pt;
+    margin-top: 25px;
+  }
+  /* Page breaks */
+  a {
+    page-break-inside: avoid;
+  }
+  blockquote {
+    page-break-inside: avoid;
+  }
+  h1,
+  h2,
+  h3,
+  h4,
+  h5,
+  h6 {
+    page-break-after: avoid;
+    page-break-inside: avoid;
+  }
+  img {
+    page-break-inside: avoid;
+    page-break-after: avoid;
+  }
+  table,
+  pre {
+    page-break-inside: avoid;
+  }
+  ul,
+  ol,
+  dl {
+    page-break-before: avoid;
+  }
+  /* Links */
+  a:link,
+  a:visited,
+  a {
+    background: transparent;
+    font-weight: bold;
+    text-decoration: underline;
+    text-align: left;
```

Comment:
This might not be a good idea for RTL.
and links are bold _and_ underlined?

Reply:
@joyously I've copied and adjusted the print.css from the Twenty Nineteen theme and you definitely have a good point regarding `text-align: left;` for RTL. 

Regarding the link, don't you think it's better to have them formatted bold and underlined to highlight that those were links?

Reply:
Whatever.... It seemed excessive to me since there is also the `href`.

Reply:
Definitely remove `text-align: left`

Reply:
I removed `text-align: left` with https://github.com/WordPress/twentytwenty/pull/97/commits/1ef9a89fa44506f0a62a021efa5bb43fcc7331a0